### ISSUE TITLE
New term proposal : isotopolog_atom_pos

### ISSUE THREAD

**Issue by mslarae13** [GenomicsStandardsConsortium/mixs#585]
**New term details**
For us to assess a new term request we require the following details:
```
Term name - Set of labeled atoms in isotopologue
Structured comment name - isotopolog_atom_pos
Definition - [a clear and concise description of the term including ]
Expected value - [e.g. text or EFO and/or OBI etc...]
Value syntax - [e.g. {float} {unit}|{termLabel} {[termID]}|{text}|{timestamp} etc...]
Example - [provide and example value]
Preferred unit - [if appropriate]
Package(s) - new checklist term applicable to all packages / extensions
```

**Additional context**
Add any other context about the new term here.

**Comment by simpso91** [comment]
Term name - Set of labeled atoms in isotopologue
Structured comment name - isotopolog_atom_pos
Definition - InChI label designating set of all isotopically enriched atoms in isotopologue
Expected value - InCHI label or "undefined"
Value syntax - {termLabel} {[termID]}|{text}
Example - 1S/C7H8/c1-7-5-3-2-4-6-7/h2-6H,1H3/i1+1,2+1,3+1,4+1,5+1,6+1,7+1  or undefined or 1S/C7H8/c1-7-5-3-2-4-6-7/h2-6H,1H3/i1+1,2+1,3+1,4+1,5+1,6+1,7+1  | undefined
Preferred unit - NA

**Comment by mslarae13** [comment]
@simpso91  can this be multi-valued? Does order matter?

**Comment by simpso91** [comment]
Yes, isotopolog_atom_pos can be multivalued and the order should be the same as isotopologue!
